### PR TITLE
chapter1_part1: /010_Intro/05_What_is_it.asciidoc

### PR DESCRIPTION
**Pull request by GavinFoo**

### CODE REVIEW

File: 010_Intro/05_What_is_it.asciidoc [hunk 1]
```diff
@@ -1,77 +1,42 @@
...
+如果你对 Elasticsearch 的功能、插件、SDK 等任何方面有疑问，都可以在这里提出 https://discuss.elastic.co[discuss.elastic.co]，http://elasticsearch.cn/[elastic 中文社区]。
+
+.回忆时光
 ***************************************
 
-Many years ago, a newly married unemployed developer called Shay Banon
-followed his wife to London, where she was studying to be a chef. While looking
-for gainful employment, he started playing with an early version of Lucene,
-with the intent of building his wife a recipe search engine.
+很多年之前，Shay Banon 刚刚结婚的失业开发者跟随他的妻子来到了伦敦，她曾在这里学习厨艺。Shay 在寻找工作的同时，还为她的妻子制作了一个配方搜索引擎——这便是 Lucene 的前身。
```
Comment (medcl) [inline 010_Intro/05_What_is_it.asciidoc]: 多年以前，有个叫 Shay Banon 的失业开发者跟随他的新婚妻子来到了伦敦，因为他妻子将在那里学习厨艺。Shay 在寻找工作的同时，开始研究还是早期版本的 Lucene ，并打算为她妻子制作一个烹饪菜谱搜索引擎。

File: 010_Intro/05_What_is_it.asciidoc [hunk 2]
```diff
@@ -1,77 +1,42 @@
...
+
+直接在 Lucene 上开发是非常困难的，所以 Shay 开始尝试将添加一个抽象层，以便于 Java 工程师可以在他们的程序中使用搜索引擎。随后，他发布了自己的第一个开源项目 “Compass”。
 
-Working directly with Lucene can be tricky, so Shay started work on an
-abstraction layer to make it easier for Java programmers to add search to
-their applications.  He released this as his first open source project, called
-Compass.
+直接基于Lucene工作会比较困难，所以Shay开始抽象Lucene代码以便Java程序员可以在应用中添加搜索功能。他发布了他的第一个开源项目，叫做“Compass”。
```
Comment (medcl) [inline 010_Intro/05_What_is_it.asciidoc]: 所以 Shay 开始实现一个 Lucene 之上的抽象层，这样Java程序员可以很方便的为他们的应用程序添加搜索功能

Comment (medcl) [inline 010_Intro/05_What_is_it.asciidoc]: 1.中英文空格问题
2.『他发布了』前面加个『于是』?

File: 010_Intro/05_What_is_it.asciidoc [hunk 3]
```diff
@@ -1,77 +1,42 @@
...
-at https://github.com/elastic/elasticsearch[github.com/elastic/elasticsearch]. See
-https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md[Contributing to 
-Elasticsearch] if you would like to join our amazing community of contributors!
-
-If you have any questions related to Elasticsearch, including specific features, 
-language clients and plugins, join the conversation at 
-https://discuss.elastic.co[discuss.elastic.co]. 
-
-.The Mists of Time
+== 了解搜索
```
Comment (medcl) [inline 010_Intro/05_What_is_it.asciidoc]: You Know, for Search 
这是一句口号，建议保留原文

File: 010_Intro/05_What_is_it.asciidoc [hunk 4]
```diff
@@ -1,77 +1,42 @@
...
+
+Elasticsearch 是一个建立在全文搜索引擎 https://lucene.apache.org/core/[Apache Lucene(TM)] 基础上的开源搜索引擎，可以说 ((("Apache Lucene")))((("Lucene")))  Lucene 是当今最先进，最高效的全功能开源搜索引擎框架。
```
Comment (medcl) [inline 010_Intro/05_What_is_it.asciidoc]: Elasticsearch 是一个建立在全文搜索引擎框架 https://lucene.apache.org/core/[Apache Lucene(TM)] 基础上的开源搜索引擎， 无论是开源软件还是私有软件，((("Apache Lucene")))((("Lucene")))  Lucene 都毫无疑问是当今最先进、性能最高和功能最全的搜索引擎框架。

File: 010_Intro/05_What_is_it.asciidoc [hunk 5]
```diff
@@ -1,77 +1,42 @@
...
+
+但是 Lucene 只是一个框架，要充分使用它的功能，你需要使用 JAVA，并且在你的程序中集成 Lucene。更糟的是，你需要做很多的了解与学习，才能明白它是如何运行的，Lucene _确实_非常复杂。
+
+Elasticsearch 也是使用 Java((("Java"))) 来编写的，并且采用了 Lucene 作为索引与搜索的内部引擎，但是在你使用它做全文搜索时，只需要使用简单流畅的 RESTful API 即可，并不需要了解 Lucene 复杂的的运行原理。
+
+当然 Elasticsearch 并不仅仅是 Lucene 那么简单，它``不仅''包括了全文搜索功能。((("Elasticsearch", "capabilities"))) 它还可以完成以下工作:
```
Comment (medcl) [inline 010_Intro/05_What_is_it.asciidoc]: ``不仅'' =》 ``不仅``

Comment (medcl) [inline 010_Intro/05_What_is_it.asciidoc]: 这一段和原意有一点出入

File: 010_Intro/05_What_is_it.asciidoc (hunk 2 above)
Comment (pengqiuyuan) [inline 010_Intro/05_What_is_it.asciidoc]: 这段中英文间没有添加空格

File: 010_Intro/05_What_is_it.asciidoc (hunk 4 above)
Comment (sunyonggang) [inline 010_Intro/05_What_is_it.asciidoc]: 第一次出现，框架这个词后面需不需要加上library，像这样框架(library)?
然后按照markdown的语法，[Apache Lucene(TM)](https://lucene.apache.org/core/)是不是更清楚些？
:runner:

File: 010_Intro/10_Installing_ES.asciidoc [hunk 6]
```diff
@@ -104,10 +77,12 @@ Elasticsearch cluster.
 --------------------------------------------------
 <1> Windows: `bin\kibana.bat plugin --install elastic/sense`.
 +
+NOTE: 如果你需要 https://www.elastic.co/guide/en/sense/current/installing.html#manual_download[在离线主机上安装 Sense]，可以直接下载 https://download.elastic.co/elastic/sense/sense-latest.tar.gz。
+
 NOTE: You can download Sense from https://download.elastic.co/elastic/sense/sense-latest.tar.gz
```
Comment (medcl) [inline 010_Intro/10_Installing_ES.asciidoc]: 多余的英文内容

File: 010_Intro/10_Installing_ES.asciidoc [hunk 7]
```diff
@@ -104,10 +77,12 @@ Elasticsearch cluster.
 --------------------------------------------------
 <1> Windows: `bin\kibana.bat plugin --install elastic/sense`.
 +
+NOTE: 如果你需要 https://www.elastic.co/guide/en/sense/current/installing.html#manual_download[在离线主机上安装 Sense]，可以直接下载 https://download.elastic.co/elastic/sense/sense-latest.tar.gz。
```
Comment (medcl) [inline 010_Intro/10_Installing_ES.asciidoc]: 这里有的不通顺，是不是换成『如果需要离线安装，你可以从这里直接下载』

File: 010_Intro/10_Installing_ES.asciidoc [hunk 8]
```diff
@@ -1,55 +1,43 @@
 [[running-elasticsearch]]
-=== Installing and Running Elasticsearch
+=== 安装并运行 Elasticsearch
 
-The easiest way to understand what Elasticsearch can do for you is to
-play with it, so let's get started!((("Elasticsearch", "installing")))
+了解 Elasticsearch 最简单的方法就是去尽情地去使用它，让我们一起开始探索之旅吧！((("Elasticsearch", "installing")))
```
Comment (medcl) [inline 010_Intro/10_Installing_ES.asciidoc]: 此处需要加"尽情"么？

**Comment by GavinFoo** [comment]
@medcl 修改过了

**Comment by medcl** [comment]
本地文件编译失败，是不是有编码问题，确认下是utf8么

**Comment by medcl** [comment]
@GavinFoo  错误信息如下：

de/510_Deployment/70_conclusion.asciidoc
asciidoc: 510_Deployment.asciidoc: line 18: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/510_Deployment/70_concla2x: ERROR: "xmllint" --nonet --noout --valid "/Users/medcl/Documents/codes/github/book.xml" returned non-zero exit status 1
usion.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment.asciidoc
asciidoc: 07_Admin.asciidoc: line 29: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/10_dynamic_settings.asciidoc
asciidoc: 520_Post_Deployment.asciidoc: line 9: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/10_dynamic_settings.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/20_logging.asciidoc
asciidoc: 520_Post_Deployment.asciidoc: line 11: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/20_logging.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/30_indexing_perf.asciidoc
asciidoc: 520_Post_Deployment.asciidoc: line 13: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/30_indexing_perf.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/40_rolling_restart.asciidoc
asciidoc: 520_Post_Deployment.asciidoc: line 15: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/40_rolling_restart.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/50_backup.asciidoc
asciidoc: 520_Post_Deployment.asciidoc: line 17: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/50_backup.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/60_restore.asciidoc
asciidoc: 520_Post_Deployment.asciidoc: line 19: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/60_restore.asciidoc
asciidoc: include: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/70_conclusion.asciidoc
asciidoc: 520_Post_Deployment.asciidoc: line 21: reading: /Users/medcl/Documents/codes/github/elasticsearch-definitive-guide/520_Post_Deployment/70_conclusion.asciidoc

a2x: executing: "xmllint" --nonet --noout --valid "/Users/medcl/Documents/codes/github/book.xml"

/Users/medcl/Documents/codes/github/book.xml:558: parser error : Input is not proper UTF-8, indicate encoding !
Bytes: 0xE3 0x22 0x3E 0x68
载： <ulink url="https://download.elastic.co/elastic/sense/sense-latest.tar.gz

**Comment by medcl** [comment]
@GavinFoo   这个 PR 能抽空处理一下么？

**Comment by tangmisi** [comment]
ref #252

**Pull request closed without merging**